pkg/controller/controller_worker_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/atlassian/smith/pkg/cleanup"
```

Comment:
make fmt doesn't reorder this? I wonder if we should just give up and have a single list of imports?